pandas/tests/io/test_sql.py

```diff
@@ -1307,9 +1312,10 @@ def check(col):
         assert is_datetime64tz_dtype(col.dtype)
         assert str(col.dt.tz) == 'UTC'
         expected = sql.read_sql_table("types_test_data", self.conn)
-        tm.assert_series_equal(df.DateColWithTz,
-                               expected.DateColWithTz
-                               .astype('datetime64[ns, UTC]'))
+        col = expected.DateColWithTz
+        assert is_datetime64tz_dtype(col.dtype)
+        # Removed ".astype('datetime64[ns, UTC]')"after GH 6415 was fixed
```

Comment:
remove this comment; you can explain why things are, but refering to the past is confusing